src/handlers/localstorage/globalSettings.js

```diff
@@ -63,3 +64,20 @@ export const saveTransactionSignatures = transactionSignatures =>
  */
 export const getTransactionSignatures = () =>
   getGlobal(TRANSACTION_SIGNATURES, {});
+
+/**
+ * @desc get dev enabled preference
+ * @param  {Boolean}  [value]
+ */
+export const getTestnetsEnabled = async preference => {
+  const test = getGlobal(TESTNETS_ENABLED, preference);
```

Comment:
can follow the pattern of the other functions and just do:
```
export const getTestnetsEnabled = async preference =>
  getGlobal(TESTNETS_ENABLED, preference);
```

Remember that `getGlobal` is an async function so line 73 might not be doing what you expect, though in this case we're just passing the promise back up